examples/L5PC/README.rst

```diff
@@ -67,6 +67,7 @@ The keys of the dictionary are the names of the models that will be built. Here,
 * ``params`` contain the essential mechanisms specifying their locations (e.g., axonal, somatic) as well as their distributions and parameters, which can be either frozen or free.
 * ``features`` contains the path to the file that includes the output of the extraction step, see `Extraction`_ for more details.
 * ``pipeline_settings`` contains settings used to configure the pipeline. There are many settings, that can each be important for the success of the model building procedure. The complete list of the settings available can be seen in the API documentation of the class `EModelPipelineSettings <../../bluepyemodel/emodel_pipeline/emodel_settings.py>`_. An important setting if you wish to run e-feature extraction through the pipeline is ``path_extract_config`` which points to the path of the json file containing the targets of the extraction process (e.g. ``L5PC_config.json``), features names, protocols and files (ephys data). More details on how to generate this file can be found in the section `Extraction`_.
+* ``morph_modifiers`` specifies morph modifiers mainly used to replace the axon in original morphology. Use a string for a named modifier or a list with ``[file path, function name, optional "hoc_string"]``. If not specified, a default modifier ``["replace_axon_with_taper"]`` is applied which will replace the axon with a tapered axon initial segment **(Note: using this modifier requires a given morphology with at least 3 axon sections.)** . Predefined modifiers can be found in the `modifiers.py <../../bluepyemodel/evaluation/modifiers.py>`_ e.g., ``["replace_axon_legacy"]``. If the provided morphology has no axon or less than three sections, set it to ``["bluepyopt_replace_axon"]``. Set it to ``[]`` to use the provided morphology without any change.
```

Comment:
There is no morph_modifiers in the example above.

Also, this is part of pipeline_settings. It is a good thing to describe morph_modifiers, but make it explicit that it is part of piepline_settings, and not something on its own.

Reply:
I will explicitly add the default value to make it clear. Actually, `morph_modifiers` can also be provided as a separate entry in the recipe (https://github.com/BlueBrain/BluePyEModel/blob/ff8768bb1df60d73efb8b8d3693fe14f43bed626/bluepyemodel/access_point/local.py#L201)